docs/book/src/developer/providers/implementers-guide/generate_crds.md

```diff
@@ -83,10 +83,8 @@ make manifests
 
 The cluster API CRDs should be further customized:
 
-- [Apply the contract version label to support conversions](https://release-0-4.cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#apply-the-contract-version-label-clusterx-k8sioversion-version1_version2_version3-to-your-crds)
-- [Upgrade to CRD v1](https://release-0-4.cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#upgrade-to-crd-v1)
-- [Set “matchPolicy=Equivalent” kubebuilder marker for webhooks](https://release-0-4.cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#add-matchpolicyequivalent-kubebuilder-marker-in-webhooks)
-- [Refactor the kustomize config folder to support multi-tenancy](https://release-0-4.cluster-api.sigs.k8s.io/developer/providers/v1alpha2-to-v1alpha3.html#refactor-kustomize-config-folder-to-support-multi-tenancy-when-using-webhooks)
+- [Apply the contract version label to support conversions](../contracts.md#api-version-labels)
+- [Set kubebuilder tags for webhooks](../webhooks.md#implementing-webhooks-with-controller-runtime-and-kubebuilder)
```

Comment:
nit, up to you.

We probably don't need the link here given that we have a subsequent section which is part of the implementer's guide anyway. Also the kubebuilder tags on webhooks don't really customize CRDs :)